### DISCUSSION TITLE
`Merchant is blocked` while testing transaction requests

### DISCUSSION THREAD

**Discussion by Henry-E** [phantom/docs#262] [Help & Support] [answered]
### Summary

While testing out transaction requests and QR code using a temporary ngrok server url, phantom has started blocking these requests saying `Error, merchant is blocked`

### Steps to Reproduce

Not sure how to help reproduce without posting a QR code or something to scan. Essentially all temporary ngrok urls which I'm using to test transaction requests are blocked.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@Henry-E Thanks for flagging! We are investigating

**Reply by adamdelphantom** [reply, marked as answer]
This appears to have been a transient simulation issue. Please let us know if you see it recur.
